build.gradle

```diff
@@ -18,8 +18,12 @@ dependencies {
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	implementation 'com.github.pengrad:java-telegram-bot-api:6.3.0'
 	implementation 'com.vdurmont:emoji-java:3.2.0'
-	implementation group: 'org.postgresql', name: 'postgresql', version: '42.5.4'
-
+	implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.0.0'
+	implementation 'org.postgresql:postgresql:42.5.1'
+	implementation 'com.h2database:h2'
+	implementation 'org.liquibase:liquibase-core'
+	// 2023/02/26, Alex: to avoid NoProviderFoundException during application start
+	implementation 'org.hibernate:hibernate-validator:6.0.13.Final'
```

Comment:
Без этой зависимости у меня при запуске приложения выдает ошибку:
"jakarta.validation.NoProviderFoundException: Unable to create a Configuration, because no Jakarta Bean Validation provider could be found. Add a provider like Hibernate Validator (RI) to your classpath."
Ну не совсем ошибку наверное, т.к. приложение запускается и работает. Но это сообщение при запуске выглядит как ошибка и раздражает. Возможно это из-за того, что у нас 'org.springframework.boot' version '3.0.2'?